api/openapi/twins.yml

```diff
@@ -1,430 +0,0 @@
-# Copyright (c) Abstract Machines
-# SPDX-License-Identifier: Apache-2.0
-
```

Comment:
Should API documentation be in one place so that it is deployed to one site or?

Reply:
Yes, I will revert that and add relevant comments.